src/components/labs/EquipmentScheduleGrid.tsx

```
@@ -0,0 +1,127 @@
+                    <div className="text-[11px] text-gray-500">{eq.labLocation}</div>
+                    {eq.requiredCertificationName && (
+                      <div className="text-[10px] text-purple-700 font-bold flex items-center gap-1 mt-0.5">
+                        <ShieldCheck size={11} /> Cert Required
+                      </div>
+                    )}
+                  </td>
+
+                  {timeSlots.map((slot, idx) => {
+                    // Check if reserved
+                    const isReserved = reservations.some(
+                      (r) => r.equipmentId === eq.id && r.startTime.includes(slot)
```

ESLint

[error] Insert `,`

(prettier/prettier)